src/main/kotlin/com/qverkk/Main.kt

```diff
@@ -83,6 +85,8 @@ enum class LoginErrorEnum(val value: Int) {
 // DATA MODELS
 // ==============================================================================
 
+val xmlMapper = XmlMapper()
```

Comment:
Note: I wrote this kinda more object oriented... so this should be in Huawei class as property and called using this.xmlMapper ?

Reply:
Oh yeah for sure, changing this

Reply:
Updated in my repo ;)